### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by aburgasser**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
 # pySpextool
 Python code to reduce data obtained with the SpeX spectrograph
 
-# Notes
-- The calibration files stored in the `instruments/` directory 
+# Installation
+The calibration files stored in the `instruments/` directory
```
Comment (kelle) [inline README.md:5]:
```suggestion
The calibration files stored in the `data/` and `instruments/` directories
```

Comment (michaelcushing) [reply README.md:5]: I am fine with this.  Except to note that currently the instruments/ directory is called instruments_data/.  I think it naturally is better called instruments/ as it contains data and code.  I was going to ask to change it in the future (I think @kelle changed it to instruments_data/ at some point in the past) but just hadn't gotten around to it.  Just making sure that it is ok for the README file to not have the exact name correct.

**Comment by kelle** [comment]
Adam, could you try this: 
`pip install -e pyspextool@git+https://github.com/pyspextool/pyspextool.git`

File: README.md (hunk 1 above)
Comment (kelle) [reply README.md:5]: I think it's fine for the README to not be perfectly accurate at this point. The point here is just to motivate the need to install `git_lfs`

**Comment by kelle** [comment]
Ok, here's the recommendation from my hive mind:

**For users:** 
```
pip install pyspextool@git+https://github.com/pyspextool/pyspextool.git
```

**For developers:**
```
git clone https://github.com/pyspextool/pyspextool.git
cd pyspextool
pip install -e ./
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+# Installation
+The calibration files stored in the `instruments/` directory 
 are stored using
 [Github Large File Storage](https://docs.github.com/en/repositories/working-with-files/managing-large-files). Please [install git-lfs](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage) before cloning this repository.
+
+## Instructions
+* clone the respository to your local computer
+>> git clone https://github.com/pyspextool/pyspextool.git
+* cd into pyspextool and install with pip/setuptools
+>> python3 -m pip install -e .
```
Comment (kelle) [inline README.md:13]:
```suggestion
**For Users**
>> pip install pyspextool@git+https://github.com/pyspextool/pyspextool.git

** For Developers**
* Ensure you are in a Python 3.9 environment and the packages listed in `[requirements.txt](https://github.com/pyspextool/pyspextool/blob/main/requirements.txt)` file are installed.
* clone the repository to your local computer
>> git clone https://github.com/pyspextool/pyspextool.git
* cd into pyspextool and install with pip/setuptools
>> cd pyspextool
>> pip install -e ./
```

**Review by kelle — CHANGES_REQUESTED** [review]
Adam, I made some suggestions. If you like them, you can commit them directly into this PR with the "Commit suggestion" button.

**Comment by kelle** [comment]
Also, when you use the -e install, you shouldn't need to re-install every time. You should be able to update the local files however you normally do (git pull) and your install should see the updated files.

**Review by kelle — APPROVED** [review]

**Pull request merged**